### PR TITLE
update Jupyter evaluation 2

### PR DESCRIPTION
**Pull request by FungluiKoo**
Updated Jupyter nbviwer evaluation

### CODE REVIEW

File: Jupyter_evaluation_2.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+---
+
+## Project Data
+
+1. Project description: <br>
+	Project Jupyter exists to develop open-source software, open-standards, and services for interactive computing across dozens of programming languages.
+
+1. Project website/homepage: [link](https://jupyter.org/)
+
+1. Project repository: [link](https://github.com/jupyter/jupyter)
```
Comment (joannakl) [inline Jupyter_evaluation_2.md]: The comments below seem to apply to a different repository: jupyter vs nbviewer

Comment (FungluiKoo) [reply Jupyter_evaluation_2.md]: Oh yes, I am evaluating a Jupyter's sub-project called nbviewer. I will do it right away. So sorry for the lateness, as I didn't see the GitHub email notification until I found it just now in my spam folder :(

**Comment by FungluiKoo** [comment]
updated Jupyter nbviwer evaluation

File: Jupyter_evaluation_2.md (hunk 1 above)
Comment (FungluiKoo) [reply Jupyter_evaluation_2.md]: Hi professor, i think I am done. Hopefully it meets the requirement now

**Pull request merged**